app/components/donation/donation-container.js

```diff
@@ -1,11 +1,24 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed: {
+    and, empty, or
+  }
 } = Ember;
 
 export default Component.extend({
   classNames: ['donation-container'],
   donationAmount: 0,
-  projectTitle: null
+  projectTitle: null,
+  wasNewCard: true,
+
+  isNewCard: empty('card.id'),
```

Comment:
@begedin can you make an issue refing this PR for us to fix? And put under this milestone, I guess.